package.json

```diff
@@ -22,6 +22,13 @@
         "url": "https://github.com/Microsoft/tslib.git"
     },
     "main": "tslib.js",
+    "exports": {
+        ".": {
+            "import": "./tslib.es6.js",
+            "default": "./tslib.js"
+        },
+        "./": "./"
```

Comment:
Do we need this one? What will break if we don't include it?

Reply:
This is mostly for `require.resolve` use cases and exactly makes this backwards compatible. For example:

* `require.resolve('tslib/tslib.d.ts')`
* `require.resolve('tslib/package.json')`
* `require.resolve('tslib/tslib.es6.js')`
* `require('tslib/tslib.js')`